### ISSUE TITLE
chrono@0.3.1 was yanked

### ISSUE THREAD

**Issue by mmun** [mongodb-labs/mongo-rust-driver-prototype#220]
Getting errors when I build:

```
error: no matching version `^0.3.1` found for package `chrono` (required by `mongodb`)
location searched: registry https://github.com/rust-lang/crates.io-index
versions found: 0.4.0, 0.3.0, 0.2.25, ...
```

More details https://github.com/chronotope/chrono/issues/156

**Comment by pengshaomin** [comment]
me to

**Comment by jjpe** [comment]
This just hit me as well. A new `mongodb` release  that relies on `chrono 0.4.0` instead would be most welcome.

**Comment by mmun** [comment]
If we want to upgrade it looks like we'll need to wait until `bson` upgrades to `chrono@0.4.0` since this library interfaces closely with `bson`. I have a patch ready for both libraries.

**Comment by jjpe** [comment]
Any idea yet when those releases will take place?

**Comment by mmun** [comment]
Not sure but I'll submit a PR to bson-rs asap :)

**Comment by mmun** [comment]
The other option, if bson-rs doesn't upgrade, is to to downgrade the deps here to chrono@0.3.0.

**Comment by jjpe** [comment]
Well I was just about to upgrade the dependencies actually, and being able to finish that properly would be nice. 
But indeed, if really necessary I could just toss the changes, no harm done.

**Comment by saghm** [comment]
Thanks for reporting! @kyeah, any chance you could merge the fix for bson once you're not busy?

**Comment by kyeah** [comment]
bson should be released on 0.9.0 — good to open a patch, @mmun (or @jjpe).

**Comment by saghm** [comment]
Fixed in #222 and published in 0.3.1. Thanks to everyone for notifying us about this issue!
